src/server.cpp

```diff
@@ -1191,12 +1204,22 @@ void CServer::MixEncodeTransmitData ( const int iChanCnt,
     else
     {
         // Stereo target channel -----------------------------------------------
+
+        const int maxPanDelay = MAX_DELAY_PANNING_SAMPLES; // 32  -->  64
+
+        int iLpan, iRpan, iPan;
+
         for ( j = 0; j < iNumClients; j++ )
         {
             // get a reference to the audio data and gain/pan of the current client
             const CVector<int16_t>& vecsData = vecvecsData[j];
+            const CVector<int16_t>& vecsData2 = vecvecsData2[j];
+
             const float             fGain    = vecvecfGains[iChanCnt][j];
-            const float             fPan     = vecvecfPannings[iChanCnt][j];
+            const float             fPan     = bDelayPan ? 0.5f : vecvecfPannings[iChanCnt][j];
+            const int iPanDel = lround( (float)(2 * maxPanDelay - 2) * (vecvecfPannings[iChanCnt][j] - 0.5f) );
+            const int iPanDelL = ( iPanDel > 0 ) ? iPanDel : 0;
+            const int iPanDelR = ( iPanDel < 0 ) ? -iPanDel : 0;
 
```

Comment:
We now have with `bDelayPan = true`:

```
    const float             fGain    = vecvecfGains[iChanCnt][j];
    const float             fPan     = bDelayPan ? 0.5f : vecvecfPannings[iChanCnt][j];
```
`fPan = 0.5`

```
    const int iPanDel = lround( (float)(2 * maxPanDelay - 2) * (vecvecfPannings[iChanCnt][j] - 0.5f) );

    const int iPanDelL = ( iPanDel > 0 ) ? iPanDel : 0;
    const int iPanDelR = ( iPanDel < 0 ) ? -iPanDel : 0;

    const float fGainL = MathUtils::GetLeftPan ( fPan, false ) * fGain;
```
`GetLeftPan` is 1 for `fPan = 0.5`, i.e, `fGainL = fGain`

```
    const float fGainR = MathUtils::GetRightPan ( fPan, false ) * fGain;

```
Same here, `fGainR = fGain`.

So we could do away with `fGainL` and `fGainR` completely and just use `fGain`. Or what am I missing?

If I am right the rest of the code could shrink a lot.